### ISSUE TITLE
Add migration guide from Create React App

### ISSUE THREAD

**Issue by leerob** [vercel/next.js#17454]
# Feature request

As a user of [x] framework/language, I'd love to see an easy path towards migrating to Next.js

## Describe the solution you'd like

Documentation around common providers and how exactly you can switch to Next.js from them. 

## Additional context

Let's start with a few of the most popular and see what others in the community would get value from. 

- Create React App
- Gatsby (https://github.com/vercel/next.js/issues/17453)
- Vanilla React (https://github.com/vercel/next.js/issues/17455)

**Comment by PeppermintNDaisies** [comment]
Hello! I would like to work on this issue, but I'm not sure what should be the final product? I found this very helpful [guide](https://www.pheuberger.com/convert-create-react-app-to-nextjs/), would it be something like this?

**Comment by leerob** [comment]
Hey @PeppermintNDaisies! This isn't very well defined, yet, but you could convert the standard `npx create-react-app` template over to Next.js and then add the PR here so we can see the diff 🙏

**Comment by PeppermintNDaisies** [comment]
Will do! Where should I place the result? In the examples folder?

**Comment by leerob** [comment]
Could you leave it as a comment here? This issue will get turn into documentation eventually 👌

**Comment by PeppermintNDaisies** [comment]
Okay sure! Like a step by step then

**Comment by leerob** [comment]
@PeppermintNDaisies Yeah, here's similar docs for migrating from Gatsby -> https://github.com/vercel/next.js/pull/17491

**Comment by PeppermintNDaisies** [comment]
Perfect! I'll give them a look

**Comment by PeppermintNDaisies** [comment]
@leerob I used your Migration guide from Gatsby as a guide, do you think this is a good start?

**Comment by leerob** [comment]
We shouldn't create documentation – yet. As mentioned previously, the first step is creating an example that converts `npx create-react-app` to Next. It would also be helpful to understand if there's an easy path for migrating an ejected CRA configuration. This research needs to happen first before docs are created 🙏

**Comment by leerob** [comment]
Heads up – I removed "good first issue" as this isn't well defined yet.

**Comment by gfortaine** [comment]
[Gradual Migration Strategies For Next.js](http://op.co.de/WfVUdF)

**Comment by balazsorban44** [comment]
This issue has been automatically locked due to no recent activity. If you are running into a similar issue, please create a new issue with the steps to reproduce. Thank you.
